src/compiler/crystal/interpreter/interpreter.cr

```diff
@@ -167,8 +167,10 @@ class Crystal::Repl::Interpreter
     compiled_def = @compiled_def
 
     # Declare or migrate local variables
-    if !compiled_def || in_pry
-      migrate_local_vars(@local_vars, meta_vars)
+    # TODO: we should also migrate variables if we are outside of a block
+    # in a pry session, but that's tricky so we'll leave it for later.
+    if (!compiled_def || in_pry)
```

Comment:
Is there a particular reason for adding these parentheses?

Reply:
Ah, ko, it's from a previous change that I later undid, but apparently not fully.

Reply:
```suggestion
    if !compiled_def || in_pry
```